deepchecks/analytics/anonymous_analytics_tracking.py

```diff
@@ -0,0 +1,30 @@
+"""Module for anonymous analytics tracking."""
```

Comment:
add here another sentence clarifying the same as inside the documentation...
(no personal data etc. will ever be sent. only a single event upon import)